packages/p2p/src/hooks/tests/useP2PExchangeRate.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useP2PExchangeRate from '../useP2PExchangeRate';
+import { APIProvider, useFetch } from '@deriv/api';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(),
+}));
+
+const mockUseFetch = useFetch as jest.MockedFunction<typeof useFetch<'website_status'>>;
+
+describe('useP2PExchangeRate Hook', () => {
+    it('should return 1 if currency is not found', () => {
+        mockUseFetch.mockReturnValue({
+            // @ts-expect-error need to come up with a way to mock the return type of useFetch
+            data: { website_status: { p2p_config: {} } },
+        });
+
+        const mock = mockStore({
+            exchange_rates: {
+                data: {
+                    base_currency: 'USD',
+                    rates: {
+                        BRL: 1.3,
+                        ZWL: 1.4,
+                        NZD: 0.0001,
+                    },
+                },
+            },
+        });
```

Comment:
could we not place these in a beforeEach or outside of the test cases then modify the values accordingly?

Reply:
https://github.com/binary-com/deriv-app/pull/9351#discussion_r1285227531